tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.10
+FROM ubuntu:21.10
```

Comment:
It isn't necessary to keep using the same version across different releases. I'd defer to others (@serathius @ptabor @spzala ) to make a decision here.

Reply:
This one got updated due to ubuntu 20.10 being EOL and failing with 
```
E: The repository 'http://security.ubuntu.com/ubuntu groovy-security Release' does not have a Release file.
``` when trying to build.

Reply:
Updating to `21.10` looks good to me.